pkgs/by-name/br/browsers/package.nix

```diff
@@ -38,21 +36,14 @@ rustPlatform.buildRustPackage rec {
     wrapGAppsHook3
   ];
 
-  buildInputs =
-    [
-      atk
-      cairo
-      gdk-pixbuf
-      glib
-      gtk3
-      pango
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      darwin.apple_sdk.frameworks.AppKit
-      darwin.apple_sdk.frameworks.CoreGraphics
-      darwin.apple_sdk.frameworks.CoreText
-      darwin.apple_sdk.frameworks.Foundation
-    ];
+  buildInputs = [
+    atk
+    cairo
+    gdk-pixbuf
+    glib
+    gtk3
+    pango
+  ];
 
```

Comment:
Package needs its own commit.

Reply:
That one's an accidental inclusion. (Just getting used to jujutsu). It'll be gone in a few minutes.